File: server/AllUsers/Usergroup.py
from abc import ABC, abstractmethod

class Usergroup(ABC):

	@property
	def username(self):
		return self._username

	@username.setter
	def user_name(self, username):
		self._user_name = username

	@property
	def password(self):
		return self._password
	
	@password.setter
	def password(self, password):
		self._password = password
	

File: server/AllUsers/Usergroup.js
class Usergroup {
  constructor() {
    this._username = "";
    this._password = "";
  }

  get username() {
    return this._username;
  }

  set username(username) {
    this._username = username;
  }

  get password() {
    return this._password;
  }

  set password(password) {
    this._password = password;
  }
}
